test.hack

```diff
@@ -0,0 +1,13 @@
+namespace Hack\GettingStarted\MyFirstProgram;
+namespace HH\Lib\Str;
+
+<<__EntryPoint>>
+function main(): void {
+
+  $x = "Apples Oranges and ";
+  $y = "Bananas";
+  splice($x, $y, 1);
```

Comment:
... and this should be `Str\splice`

Reply:
good feedback! was just messing around and accidentally added when I wasn't paying attention -- removed.